modules/updating-troubleshooting-clear.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/updating-troubleshooting-clear.adoc
```

Comment:
I don't see this assembly in the PR. Seems like it should be:
```suggestion
// * updating/troubleshooting_updates/recovering-update-before-applied.adoc
```

Reply:
forgot to update after renaming :) good catch! ty